### ISSUE TITLE
disable no-multi-str

### ISSUE THREAD

**Issue by NodeGuy** [ricardofbarros/linter-js-standard#117]
I'm being warned that multiline support is limited to browsers supporting ES5 only. (no-multi-str)

Given that we're all transitioning to ES6 this isn't a big worry of mine, plus I do most of my development in Node.js anyway.  Shall we turn off this warning for everyone?

**Comment by ricardofbarros** [comment]
Hey @NodeGuy, tell me if I got this right, you want me to disable a specific rule of `standard`/`semistandard`/etc.. ? That would beat the purpose of `standard` style, no ?

**Comment by NodeGuy** [comment]
I read the [Standard rules](https://github.com/feross/standard/blob/master/RULES.md#javascript-standard-style) and I see no mention of prohibiting multiline strings so no, this would not be disabling a specific rule of `standard`.  It appears to be a vestigial, incidental warning for outdated versions of JavaScript.  It's trigged by any multiline string like this:

```JavaScript
var foobar = 'This is a multiline \
string that is allowed by JavaScript Standard Style \
but warned about by the linter.'
```

**Comment by ricardofbarros** [comment]
@NodeGuy the linter doesn't do anything more than the standard style, so if standard doesn't warn you , linter shouldn't warn as well. Could you tell me which version do you have, what are the settings, etc.. ?

**Comment by ricardofbarros** [comment]
I have just tested that sample and it didn't warn me about `no-multi-str`. Care to check the standard version you have under your project and `linter-js-standard` ?

**Comment by NodeGuy** [comment]
Hmm.

<img width="756" alt="screen shot 2016-05-26 at 11 57 55 am" src="https://cloud.githubusercontent.com/assets/2148537/15581165/3b70496c-2339-11e6-910f-19ebd246ddc6.png">

<img width="650" alt="screen shot 2016-05-26 at 11 58 36 am" src="https://cloud.githubusercontent.com/assets/2148537/15581171/3fddebee-2339-11e6-9ea8-1bbd006f4e71.png">
